### ISSUE TITLE
Staff Commands (tl;dr at bottom)

### ISSUE THREAD

**Issue by trentdunford** [sirsavary/mordrum-issues#1]
July 6, 2015

Its a warm Monday night on Mordrum. With all of this great weather comes a plentiful load of entities. Lag is apparent. SirSavary spontaneously appears online to save the day. He says, "There's a shitload of entities, wtf. Trent, you are our last and only hope. Since you've been gone shits hit the fan, everything has gone down hill rapidly. Thank goodness you are here to save us all. What should I do Trent?". In response, Trent tells sav to reboot the server and stop being a filthy mexican jew. Sav reboots the server and Spigot fucks up, causing a couple more issues to pop up but eventually everything is back to normal. Or so it appears. Trent attempts to /fly, it doesnt work. Trent attempts to /op, he is declined again. Trent, in a final attempt. types /ch join staff. After giving everything he could, he is rejected yet a third time. 

tl;dr - STAFF ARE MISSING THEIR STAFF CMDS PLS FIX.
